package.json

```diff
@@ -34,46 +34,43 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.4.2",
+    "broccoli-asset-rev": "^2.4.5",
     "chai-jquery": "^2.0.0",
-    "ember-blog-engine": "git://github.com/srowhani/ember-blog-engine.git#caf6e66089b9639cbb076b5eeaa656161f91a5b9",
```

Comment:
@srowhani @sglanzer Do either of you recall the reason for including this package and setting the dummy app up to use it?  What was trying to be accomplished?